src/components/ArtworkGridOnView.tsx

```diff
@@ -2,20 +2,20 @@ import React from 'react';
 import Observer from 'react-intersection-observer';
 
 interface Props {
-  // Called when the sentinel comes into the view.
-  onStartLoading: () => void;
+  // Called when the component comes into view.
+  onView: () => void;
 }
 
 // Calls a method on props when enters view. Often used to implement
 // infinite scrolling.
-export const ArtworkGridLoadMoreSentinel = (props: Props) => (
+export const ArtworkGridOnView = (props: Props) => (
```

Comment:
This isn't specific to artwork, it can be called something more general like `OnEnterView`